Feature/v2

corwinc opened the pull request:
Includes:
- How it Works section
- Jurisdictions (map) section

UnsungHero97 commented on src/containers/jurisdictions/JurisdictionLabel.js:
```diff
@@ -0,0 +1,83 @@
+    flex-basis: 20%;
+  }
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_LG}px) {
```
Check your logic here. `props.x` and `props.y` are undefined.

![screen shot 2018-06-25 at 10 00 04 am](https://user-images.githubusercontent.com/440299/41864400-bc35340a-785e-11e8-956b-abe45c45c75c.png)

corwinc replied:
Fixed

UnsungHero97 commented on src/containers/howitworks/Content01.js:
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import Image01 from '../../assets/images/01_problem_definition.png';
```
Please use a more descriptive name.

corwinc replied:
Fixed

UnsungHero97 commented on src/containers/howitworks/Content02.js:
```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import Image02 from '../../assets/images/02_data_sharing_agreements.png';
```
Please use a more descriptive name.

corwinc replied:
Fixed

UnsungHero97 commented on src/containers/howitworks/Content03.js:
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import Image03 from '../../assets/images/03_establishing_data_access.png';
```
Please use a more descriptive name.

corwinc replied:
Fixed

UnsungHero97 commented on src/containers/howitworks/Content04.js:
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import Image04 from '../../assets/images/04_specific_services.png';
```
Please use a more descriptive name.

corwinc replied:
Fixed

UnsungHero97 commented on src/containers/howitworks/HowItWorksSection.js:
```diff
@@ -0,0 +1,48 @@
+  }
+`;
+
+const HowItWorksSection = () => (
```
The "How It Works" section looks pretty bad when you resize the screen to below 1020px. The image basically becomes full screen.

![screen shot 2018-06-25 at 10 05 15 am](https://user-images.githubusercontent.com/440299/41864566-4d77ac2c-785f-11e8-810b-0c7f68f70ac9.png)

corwinc replied:
Fixed, but probably needs some design in put from Paul (both small and med browser sizes, and for everything)

UnsungHero97 commented on src/containers/jurisdictions/JurisdictionLabel.js:
```diff
@@ -0,0 +1,83 @@
+  display: none;
+  font-weight: normal;
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_MD}px) {
```
The population label disappears when the screen drops below 900px.

![screen shot 2018-06-25 at 10 05 57 am](https://user-images.githubusercontent.com/440299/41864639-8680a41a-785f-11e8-8260-06ec5cda10f9.png)

corwinc replied:
Fixed

corwinc commented on src/containers/howitworks/TabDisplay.js:
```diff
@@ -0,0 +1,107 @@
+            { TABS[4] }
+          </TabItem>
+        </Navigation>
+        { this.renderContent(this.state.tab) }
```
Fixed

corwinc commented:
Fixed everything related to your comments. Also includes:
- Merged develop and fixed related lint errors
- Fixed inaccurate line heights
- Added links as needed to How it Works section
- Renamed style.js to Style.js
